main/adapters/database/database.go

```diff
@@ -62,16 +63,19 @@ func NewDatabaseManager(driverName, dataSourceName string, maxConns int) (*Datab
 		return nil, fmt.Errorf("empty database driverName or dataSourceName")
 	}
 
-	var isolationLvl sql.IsolationLevel
-	var createStatements []string
+	dm := &DatabaseManager{}
 
 	switch driverName {
 	case PostgreSQL:
-		isolationLvl = sql.LevelReadCommitted
-		createStatements = createPostgres
+		dm.driverName = PostgreSQL
```

Comment:
I believe that these details could be push back to outer layers. But as mentioned, this is not the scope for this PR.